packages/react-router/modules/index.js

```diff
@@ -7,3 +7,4 @@ export StaticRouter from './StaticRouter'
 export Switch from './Switch'
 export matchPath from './matchPath'
 export withRouter from './withRouter'
+export { resolveLocation } from './resolve'
```

Comment:
While it's not yet used anywhere else, any reason to not export `resolvePath` too? Seems like it could be useful.